example.ipynb

```diff
@@ -213,1997 +404,504 @@
+       " 'Hepatocellular Carcinoma ',\n",
+       " 'Lung Adenocarcinoma',\n",
+       " 'Lung Squamous Cell Carcinoma',\n",
+       " 'Oral Squamous Cell Carcinoma',\n",
+       " 'Other',\n",
+       " 'Ovarian Serous Cystadenocarcinoma',\n",
+       " 'Pancreatic Ductal Adenocarcinoma',\n",
+       " 'Papillary Renal Cell Carcinoma',\n",
+       " 'Pediatric/AYA Brain Tumors',\n",
+       " 'Rectum Adenocarcinoma',\n",
+       " 'Uterine Corpus Endometrial Carcinoma']"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "unique_terms('ResearchSubject.primary_disease_type', system=\"PDC\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "continued-immigration",
+   "metadata": {},
+   "source": [
+    "After examining the output, we see that it does come from the PDC. Hence, if we could first identify the data that has research subjects found within the PDC that have this particular disease type, and then further narrow down the results to include only the portion of the data that is present in GDC, we could get the records that we are looking for."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "speaking-ethernet",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Query: SELECT * FROM (SELECT * FROM gdc-bq-sample.cda_mvp.v3, UNNEST(ResearchSubject) AS _ResearchSubject, UNNEST(_ResearchSubject.identifier) AS _identifier WHERE ((_ResearchSubject.primary_disease_type = 'Ovarian Serous Cystadenocarcinoma') AND (_identifier.system = 'PDC'))), UNNEST(ResearchSubject) AS _ResearchSubject, UNNEST(_ResearchSubject.identifier) AS _identifier WHERE (_identifier.system = 'GDC')\n",
+      "Offset: 0\n",
+      "Limit: 1000\n",
+      "Count: 275\n",
+      "More pages: No\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "q1 = Q('ResearchSubject.primary_disease_type = \"Ovarian Serous Cystadenocarcinoma\"')\n",
+    "q2 = Q('ResearchSubject.identifier.system = \"PDC\"')\n",
+    "q3 = Q('ResearchSubject.identifier.system = \"GDC\"')\n",
+    "\n",
+    "q = q3.From(q1.And(q2))\n",
+    "r = q.run()\n",
+    "\n",
+    "print(r)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "celtic-garage",
+   "metadata": {},
+   "source": [
+    "As you can see, this is achieved by utilizing ```From``` operator. The ```From``` operator allows us to create queries from results of other queries. This is particularly useful when working with conditions that involve a single field which can take multiple different values for different items in a list that is being part of, e.g. we need ```ResearchSubject.identifier.system``` to be both “PDC” and “GDC” for a single patient. In such cases, ```And``` operator can’t help because it will return those entries where the field takes both values, which is zero entries."
```

Comment:
I think this text uses curly double quotes, not sure if those render correctly everywhere.

This is a good explanation! I think in addition to being here, we should have another top level doc that's only the text about the APIs and usage. Something to work on in the upcoming weeks.

Reply:
I have no idea where to look for those curly double quotes 👀

Reply:
I agree that we'll need more documentation. Let's keep that under the radar.